rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp

```diff
@@ -5207,11 +5207,11 @@ void Compiler::CreateBranch(llvm::Value * cmp_i1, llvm::Value * target_i32, bool
 			}
 
 			SetPc(target_i32);
-//			Function *fn = m_module->getFunction(fmt::format("function_0x%08X", target_address));
+			Function *fn = m_module->getFunction(fmt::format("function_0x%08X", target_address));
```

Comment:
@tambry 
Yes, I just wanted tested, and since I saw no problem, I thought it had been turned off long ago for an old llvm.
Should that have known who was already disabled.